Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -87,6 +88,20 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 	bool bExpectedActorCountsInitialised;
 	int32 ActivePlayers; // All authoritative players from all workers
 
+	// For actor migration count
+	bool bHasActorMigrationCheckFailed;
+	int32 PreviousTickMigration;
+	typedef TTuple<int32, float> ToBeRemovedDelta;
```

Comment:
Is this typedef necessary? At the least change the type name, MigrationDeltaPair (for example), Generally speaking naming things ToBeXXXXX feels clumsy, I prefer naming containers based on what they contain, not how they will be used.. e.g. MigrationDeltas ... if it's necessary to decribe the usage a comment should suffice

Reply:
There are 2 places to use the type TTuple<int32, float>, so I make a typedef.
I will change the name to MigrationDeltaPair